internal/service/project/resource_project.go

```diff
@@ -661,7 +663,7 @@ func GetProjectPropsFromAPI(ctx context.Context, projectsAPI admin.ProjectsApi,
 }
 
 func SetSlowOperationThresholding(ctx context.Context, performanceAdvisorAPI admin.PerformanceAdvisorApi, projectID string, enabledPlan types.Bool) error {
-	if enabledPlan.IsNull() {
+	if enabledPlan.IsNull() || enabledPlan.IsUnknown() {
 		return nil
 	}
 	enabled := enabledPlan.ValueBool()
```

Comment:
test case to cover here with different keys: when user does not have permissions, it throws an error

Reply:
follow-up ticket for tests: CLOUDP-280722